File: C atau C++/kasirmini.cpp
#include <iostream>
#include <iostream>
using namespace std;

int main() {
    string nama;
    float harga;
    int jumlah;
    float total, diskon, total_bayar;

    cout << "Masukkan nama barang: ";
    getline(cin, nama)
    cout << "Masukkan harga barang: ";
    cin >> harga;
    cout << "masukkan jumlah beli: ";
    cin >> jumlah;

    total = harga * jumlah;

    if (total > 65100) {
        diskon = total * 0.01;
    } else {
        diskon = 0;
    
    }

    total_bayar = total - diskon;

    cout << "\n====== STRUK PEMBELIAN ======\n";
    cout << "Nama Barang : " << nama << "\n";
    cout << "Harga Satuan : " << harga << "\n";
    cout << "Jumlah Beli : " << jumlah << "\n";
    cout << "Total Harga : " << total << "\n";
    cout << "Diskon : " << diskon << "\n";
    cout << "Total Bayar : " << total << "\n";
    
    return 0;
}

File: php/kasirmini.php
<?php
echo "Masukkan nama barang: ";
$nama = trim(fgets(STDIN));

echo "Masukkan harga barang: ";
$harga = floatval(trim(fgets(STDIN)));

echo "Masukkan jumlah beli: ";
$jumlah = intval(trim(fgets(STDIN)));

$total = $harga * $jumlah;

if ($total > 65100) {
    $diskon = $total * 0.01;
} else {
    $diskon = 0

}

$total_bayar = $total - $diskon;

echo "\n===== STRUK PEMBELIAN =====\n";
echo "Nama Barang : $nama\n";
echo "Harga Satuan : $harga\n";
echo "Jumlah Beli : $jumlah\n";
echo "Total Harga : $total\n";
echo "Diskon : $diskon\n";
echo "Total Bayar : $total\n";
echo "=============================\n";

?>
